GolemLib/net.cs

```diff
@@ -0,0 +1,12 @@
+namespace GolemLib;
+
+public enum RelayType
+{
+    // yacn2.dev.golem.network:7477
+    Public,
+    // yacn2a.dev.golem.network:7477
+    Devnet,
+    // 127.0.0.1:16464
+    Local,
+    Central,
```

Comment:
`None` option has been removed but it is needed if a custom one should be used

Reply:
What does custom one mean?

Reply:
This has been added on purpose - the fact that this is not handled anywhere is its feature b/c the RelayType param is a required param in multiple functions then it is easier to have it accept a value of `None` that is not handled than to make it nullable and handle it somehow